### DISCUSSION TITLE
Differences between themes in different OS when using DataGrid

### DISCUSSION THREAD

**Discussion by GrigoryevArtem-torro** [AvaloniaUI/Avalonia#17378] [Q&A]
Hello!
I am trying to build UI with DataDrid component in macOS.
I am using Fluent theme in general. And I am using Fluent theme for DataGrid too. For some reason, my colleague who is working in Windows has completely different Datagrid view with many things failing. When I switch DataGrid style from Fluent to Simple:

`<StyleInclude Source="avares://Avalonia.Controls.DataGrid/Themes/Fluent.xaml" />`
changed to 
`<StyleInclude Source="avares://Avalonia.Controls.DataGrid/Themes/Simple.xaml" />`
our views synchronised.

What can be the reason for this behaviour?

Kind regards,
Artem

**Comment by stevemonaco** [comment]
Controls are expected to look different in different themes. The core control functionality remains the same, but any custom styling you have won't necessarily be compatible across themes.

You'll need to be more detailed with the "many things failing" for a better answer.
